Edited FinancialAssistance php file

MjBasco opened the pull request:
Added code and images for Financial Assistance initial look. I also added financial assistance css file filename: FAStyle.css

MjBasco committed:
Edited FinancialAssistance php file

Added code and images for Financial Assistance initial look. I also added financial assistance css file filename: FAStyle.css

Christiann12 reviewed (changes requested):
Make columns be on top of each other when the screen is small

MjBasco committed:
Merge branch 'main' into Issue-37

MjBasco committed:
Edited: Financial Assistance php file

changed buttons sizes/placement when resizing windows.

MjBasco committed:
Update FinancialAssistance.php

MjBasco committed:
Merge branch 'main' into Issue-37

MjBasco committed:
Update homepage.css

MjBasco committed:
Merge branch 'main' into Issue-37

MjBasco committed:
Edited: Financial Assistance PHP and CSS file

- Added link to Financial Assistance
  CSS file in header PHP file.
- Added media query to CSS file
- Fixed Margins and columns of
  Financial Assistance page.
- Fixed errors/mistakes in homepage
  (finpreview button color, finpreview
  size.).
- Changed company1.jpeg to
   company2.png.

Christiann12 committed:
Merge branch 'main' into Issue-37

Christiann12 reviewed (approved):
Work's perfectly
